### PR TITLE
V1.1 Rust

### PR DESCRIPTION
**Pull request by ErisMik**
After review I'll push packages and updated the dependency files before merge

### CODE REVIEW

File: binding/rust/tests/cheetah_tests.rs [hunk 1]
```diff
@@ -53,7 +54,7 @@ mod tests {
 
         assert_eq!(
             result,
-            "MR QUILTER IS THE APOSTLE OF THE MIDDLE CLASSES AND WE ARE GLAD TO WELCOME HIS GOSPEL"
+            "Mr quilter is the apostle of the middle classes and we are glad to welcome his gospel"
```
Comment (ksyeo1010) [inline binding/rust/tests/cheetah_tests.rs:57]: can we add test that does punctuation as well?

**Review by ksyeo1010 — APPROVED** [review]

**Comment by ksyeo1010** [comment]
Can you change to this readme?
```
Cheetah is an on-device streaming speech-to-text engine. Cheetah is:
- Private; All voice processing runs locally.
- [Accurate](https://picovoice.ai/docs/benchmark/stt/)
- [Compact and Computationally-Efficient](https://github.com/Picovoice/speech-to-text-benchmark#rtf)
- Cross-Platform:
    - Linux (x86_64), macOS (x86_64, arm64), and Windows (x86_64)
    - Android and iOS
    - Chrome, Safari, Firefox, and Edge
    - Raspberry Pi (4, 3) and NVIDIA Jetson Nano
```

**Comment by ErisMik** [comment]
> 

Updated

**Comment by ErisMik** [comment]
Also, packages released, ready for merge!

**Pull request merged**